manage.py

```diff
@@ -3,9 +3,10 @@
 import sys
 
 if __name__ == "__main__":
+    os.environ.setdefault("DDM_CONTAINER_NAME", "app")
     os.environ.setdefault("DJANGO_SETTINGS_MODULE", "usaspending_api.settings")
     try:
-        from django.core.management import execute_from_command_line
+        from docker_django_management import execute_from_command_line
```

Comment:
This sets up a convenient shortcut that allows `manage.py` to be used directly from the Docker Host without having to prefix it with `docker-compose run app`.  However, the shortcut relies on `import django` raising an `ImportError` on the Docker host, which is usually but not always the case, which is why we still document everything using the long-form notation.